.github/workflows/release.yaml

```
@@ -35,9 +33,10 @@ jobs:
       - name: Upload coverage report
         uses: actions/upload-artifact@v2
         with:
-          name: Coverage Report
+          name:
+            Coverage Report (Node ${{ matrix.node-version }} on ${{ matrix.os
+            }})
```

Kind of a nitpick, but I'd prefer this to be all on one line:

```suggestion
          name: Coverage Report (Node ${{ matrix.node-version }} on ${{ matrix.os }})
```